packages/cli/src/commands/database/config.ts

```diff
@@ -146,9 +149,17 @@ const rotateConfig: CommandModule<unknown, { key: string; tenantId: string }> =
         describe: 'The tenant to operate',
         type: 'string',
         default: defaultTenantId,
+      })
+      .option('type', {
+        describe: `The key type for ${
+          LogtoOidcConfigKey.PrivateKeys
+        }, one of ${validPrivateKeyTypes.join(', ')}`,
+        type: 'string',
+        default: 'ec',
       }),
-  handler: async ({ key, tenantId }) => {
+  handler: async ({ key, tenantId, type }) => {
     validateRotateKey(key);
+    const parsedType = z.enum(validPrivateKeyTypes).parse(type);
```

Comment:
seems in other scenarios, we use consoleLog.warn to show error message, should we align the logic here? Not sure how this unsafe parser show error messages.

Reply:
updated